src/compile/compile.ts

```diff
@@ -48,7 +48,7 @@ export function compileRootGroup(model: Model) {
   const spec = model.spec();
 
   let rootGroup:any = extend({
-      name: spec.name ? spec.name + '-root' : 'root',
+      name: spec.name ? spec.name + '-rootCell' : 'rootCell',
```

Comment:
I think scene in Vega sounds right to me :)


On Wed, Feb 24, 2016 at 11:44 AM Arvind Satyanarayan <
notifications@github.com> wrote:

> In src/compile/compile.ts
> <https://github.com/vega/vega-lite/pull/1176#discussion_r53995170>:
>
> > @@ -48,7 +48,7 @@ export function compileRootGroup(model: Model) {
> >    const spec = model.spec();
> >
> >    let rootGroup:any = extend({
> > -      name: spec.name ? spec.name + '-root' : 'root',
> > +      name: spec.name ? spec.name + '-rootCell' : 'rootCell',
>
> I had forgotten about this myself. Perhaps the root mark in Vega should be
> named scene instead to correspond to the scene property we introduced
> recently. Hmm.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/vega/vega-lite/pull/1176/files#r53995170>.
>